src/Illuminate/Support/Arr.php

```diff
@@ -38,6 +38,28 @@ public static function add($array, $key, $value)
         return $array;
     }
 
+    /**
+     * Returns true if any element of the array is truthy. If not, returns false.
+     *
+     * @param  array  $array
+     * @param  callable|null  $callback
+     * @return bool
+     */
+    public static function any($array, callable $callback = null)
+    {
+        $callback = $callback ?: function ($value) {
```

Comment:
Shouldn't the `value()` helper function be used here?
https://github.com/illuminate/support/blob/a2205e98eec136752e8845ca47ce249a8429fa03/helpers.php#L520-L523

Otherwise it will just return a closure from a closure without executing it if you have an array of/with closures.